src/GitHubStatusDisplay.js

```diff
@@ -813,7 +818,9 @@ export default class BuildHistoryDisplay extends Component {
       }
 
       function renderPullRequestNumber(comment) {
-        let m = comment.match(/\(#(\d+)\)/);
+        let m = comment.match(/\(#(\d+)\)/) || comment.match(
+          /https:\/\/github.com\/pytorch\/pytorch\/pull\/(\d+)/
+        );
```

Comment:
Can you please explain how this change is relevant to the goal of displaying more PRs?

Reply:
I am using the associated function to generate some Fragment.  I noticed that the code Fragments following each match were identical and combined the match functions to collapse the duplicated Fragment generation.